coordinator/coordinator.go

```diff
@@ -276,24 +270,32 @@ func (c *Coordinator) WatchForNewJobs() error {
 	c.Logger.Info("Watching for new jobs")
 	// TODO: change it so we can watch for new jobs using task manager; searching for PENDING status tasks
 	// JOB__TYPE__NAME__VARIANT
-	runs, err := c.taskManager.GetAllTaskRuns()
+	runs, err := c.Metadata.Tasks.GetAllRuns()
 	if err != nil {
 		return fferr.NewInternalError(err)
 	}
-	runs.FilterByStatus(scheduling.Pending)
+
+	runs.FilterByStatus(scheduling.PENDING)
 
 	for _, run := range runs {
```

Comment:
What's the difference between both of these loops? We only need one right?

If not, please add a comment or break into functions so that we know what's going on

Reply:
garbage that we're going to remove